### DISCUSSION TITLE
第3节课笔记(16班)

### DISCUSSION THREAD

**Discussion by Yang-Changhui** [InternLM/Tutorial#168] [笔记]
16班第3节课笔记在下面提交

**Comment by Aorg** [comment]
https://github.com/Aorg/Assignment/blob/main/langchain/lesson3_langchain_assignment.md

**Comment by future-2002** [comment]
https://blog.csdn.net/Y2483490891/article/details/135434150?spm=1001.2014.3001.5501

**Comment by Aorg** [comment]
https://github.com/Aorg/Assignment/blob/main/langchain/lesson3_langchain_note.md

**Comment by Eadral** [comment]
https://www.yuque.com/eadral/ruaugi/za2ehbdvb8tklbch?singleDoc# 《第三节课作业》

**Comment by Durian-1111** [comment]
**Day03-LearningNote**
链接：https://xby48gs45u.feishu.cn/docx/Zmx5dyhucoTnIbxtlT0cFl1Dnnb?from=from_copylink

**Comment by Aitejiu** [comment]
https://www.cnblogs.com/aitejiu/p/17956088#%E9%83%A8%E7%BD%B2-web-demo

**Comment by Star-cre** [comment]
b站公开笔记：https://www.bilibili.com/video/BV1sT4y1p71V/?vd_source=2fb638751797274bd22bea982387a179
csdn：https://zhilengnuan.blog.csdn.net/article/details/134491154?spm=1001.2014.3001.5502

**Comment by 2404589803** [comment]
https://2404589803.github.io/My-datawhale-learning/#/shusheng_puyu_learnning53/ch03

**Comment by Dhaizei** [comment]
http://t.csdnimg.cn/BDyJ3

**Comment by xuanxiaochenmi** [comment]
https://github.com/xuanxiaochenmi/InternLLM_notes/blob/master/lesson_3/lesson_3.md

**Comment by etheral12138** [comment]
https://y1yc0dwhb0o.feishu.cn/docx/KfVHdtDL9oqRGkxAc39cf0hUnNe?from=from_copylink

**Comment by morcake** [comment]
https://blog.51cto.com/morcake/9186343

**Comment by EnableAsync** [comment]
https://enableasync.github.io/internlm/internlm-03/

**Comment by hhhanqian** [comment]
94 【书生·浦语大模型笔记（三） - 嗨嗨 | 小红书 - 你的生活指南】 😆 kfwseP4FzMKqZ9E 😆 http://xhslink.com/rGrQcz

**Comment by zszbyzsz** [comment]
https://juejin.cn/post/7322275119594176575

**Comment by TianCai19** [comment]
https://codetwice.notion.site/fd98aa51b1494ef4be540fdec148e0ab
